### ISSUE TITLE
Some sounds should be prevented to be triggered two times

### ISSUE THREAD

**Issue by RastaTux** [Devp00l/pnp-soundboard#8]
It happened to me that accidently I triggered a background music twice, so it is playing two times with a slight delay.
This really sounds bad and you need to stop both and replay once, what makes an ugly cut in the experience (Since they are also not fadeable).

Instead it could be cool to distinguish a sound into background music or sound effects. The first one can't be triggered more than once. Sound effects on the other hand might be needed to get triggered more than once while the first trigger still plays.

An alternative might be that you can choose between multi trigger or one time play.

**Comment by Devp00l** [comment]
Yeah I get your point, but sometimes you want that the background music will repeat it self, so you can do that by pressing control before pressing the sound key, this would repeat the sound if the soundfile has finished.

I know my comments are a bit late and I very happy to have (had?) such an active user :)

The repeating would go endlessly until you stop all sounds  with backspace or escape.

The thing that a sound can be played multiple times before it has finished is a also needed feature, for gunshots or sword noises.
